test/case/ietf_interfaces/igmp_vlan/test.py

```diff
@@ -1,53 +1,54 @@
 #!/usr/bin/env python3
-#
-#
-#            VLAN55             VLAN77                          VLAN55              VLAN77
-#           10.0.1.1           10.0.2.1                       10.0.1.2            10.0.2.2
-#             \                    /                           \                     /
-#              \                  /                             \                   /
-#               \                /                               \                 /
-#                \--------------/        VLAN 1,2 T               \---------------/
-#                |    DUT1      +---------------------------------+      DUT2     |
-#                |              |                                 |               |
-#                +--------------+                                 +-----+---------+
-#        VLAN55 U|      VLAN77 U|                              VLAN55 U |        | VLAN77 U
-#                |              |                                       |        |
-#                |              |                                       |        |
-#                |              |                                       |        |
-#                |              |                                       |        |
-#     +-------+  |             +----------+                 +------------+       +--------+
-#     | msend +--+             | mreceive |                 | mreceive   |       | msend |
-#     +-------+                +----------+                 +------------+       +--------+
-#      10.0.1.11                 10.0.2.11                      10.0.1.22         10.0.2.22
-#
-
 """
 IGMP VLAN
 
 Test tagged IGMP control traffic and that VLAN separation is respected for multicast
+
+....
+
+       VLAN55             VLAN77                          VLAN55              VLAN77
+      10.0.1.1           10.0.2.1                       10.0.1.2            10.0.2.2
+        \                    /                           \                     /
+         \                  /                             \                   /
+          \                /                               \                 /
+           \--------------/        VLAN 1,2 T               \---------------/
```

Comment:
Not your doing (its my from the beginning), but show the ports,  dut1:<port> dut2:<port> etc

Reply:
fixed